uber/site_sections/summary.py

```diff
@@ -4,25 +4,42 @@
 @all_renderable(c.STATS)
 class Root:
     def index(self, session):
-        attendees, groups = session.everyone()
-        count = lambda *args, **kwargs: len([a for a in attendees
-                                               if all(matcher(a) for matcher in args) and
-                                                  all(val == getattr(a, name) for name, val in kwargs.items())])
```

Comment:
As above, we're iterating over the list of attendees once instead of a separate time for every count/statistic.  More code, but this still seems fairly clean and is way faster.